code/modules/games/tarot.dm

```diff
@@ -168,3 +169,220 @@
+				with spirits, madness."
+			if("Crown of Ice")
+				P.desc = "A throne of cracked glacier-ice, on it sitting a crown of deep blue ice. There are two sockets embedded in the crown, one with a sapphire, one empty. It is associated with: harsh judge, death's herald."
+			if("Dreaming Girl")
+				P.desc = "A young girl sleeping in a snowdrift, untouched by cold. Above her, the night sky swirls with auroras forming vague shapes. Her expression is peaceful, but a tear of blood escapes one eye. It is associated with: \
+				prophecy, confusion, hidden truths."
+			if("Flamewalker")
+				P.desc = "A cloaked figure walking across an icefield. He's burning. Fire doesn’t consume him— it emerges from his footprints. He carries a bundle of books and bones wrapped in red cloth. It is associated with: \
+				change-bringing, exile, innovation."
+			if("Tall Stranger")
+				P.desc = "A faceless, well-built and tall standing figure in fine red-gilded robes stands at atop a snowdrift. He holds a sword in one hand, the other's clenched to a fist. No footprints are left where he stands. \
+				It is associated with: deception, fate disguised, dishonesty."
+			if("Mother")
+				P.desc = "A woman cradling a baby in one arm, a ceremonial blade in the other. Her expression is conflicted. Behind her, two paths diverge—one leads to a firelit camp, the other into a storm. It is associated with: \
+				choices, nurture vs. law, duality."
+			if("Hollowed Man")
+				P.desc = "A man torso unclothed, kneels in the snow. A hole where his heart should be reveals a swirling void inside. Spirits drift around him. It is associated with: sacrifice, emptiness, spiritual rebirth."
+		P.card_icon = "spirits"
+		P.back_icon = "card_back_lyodii"
+		cards += P
+
+/obj/item/deck/tarot/lyodii/paths
+	name = "lyodii fatesayer paths deck"
+	desc = "A traditionally made deck of fatesayer cards, used by the people of the Lyod. This stack contains the Paths cards. "
+	icon_state = "lyodii_deck"
+	item_state = "lyodii_deck"
+
+/obj/item/deck/tarot/lyodii/paths/generate_deck()
+	var/datum/playingcard/P
+	for(var/name in list("Frozen Footprint","Shared Fire","Broken Edict","Silent Hunt","Lost Tribe","Marked Bone","Buried Blade","Icebound Oath"))
+		P = new()
+		P.name = "[name]"
+		switch(name)
+			if("Frozen Footprint")
+				P.desc = "A single footprint frozen solid in a cracked icy path. It glows faintly, untouched by time. Around it, scattered is a bloodied cloth and a broken spearhead. It is associated with: missed opportunity, consequences."
+			if("Shared Fire")
+				P.desc = "everal figures gather around a fire in the middle of a blizzard. Though cold rages around them, their circle is warm. Shadows on the tent walls show animals instead of people. One figure extends a \
```

Comment:
```suggestion
				P.desc = "Several figures gather around a fire in the middle of a blizzard. Though cold rages around them, their circle is warm. Shadows on the tent walls show animals instead of people. One figure extends a \
```
Missing "S"